packages/yourloops/pages/hcp/team-member-add-dialog.tsx

```diff
@@ -101,44 +86,47 @@ function AddMemberDialog(props: AddMemberDialogProps): JSX.Element | null {
       id="team-add-member-dialog"
       open={addMember !== null}
       aria-labelledby={t("aria-team-add-member-dialog-title", { teamName })}
-      onClose={handleClickClose}>
+      onClose={handleClickClose}
+      fullWidth
+    >
       <DialogTitle id="team-add-member-dialog-title">
         <strong>{t("team-add-member-dialog-title")}</strong>
         <br />
         <span id="team-add-member-dialog-title-team-name">{teamName}</span>
       </DialogTitle>
 
-      <DialogContent className={classes.dialogContent}>
-        <TextField
-          id="team-add-member-dialog-field-email"
-          variant="outlined"
-          onChange={handleChangeEMail}
-          name="email"
-          value={email}
-          label={t("email")}
-          required={true}
-          aria-required="true"
-          type="email"
-        />
-        <FormControlLabel
-          control={
-            <Checkbox
-              id="team-add-member-dialog-checkbox-admin"
-              checked={role === TeamMemberRole.admin}
-              onChange={handleChangeRole}
-              name="role"
-              color="primary"
-            />
-          }
-          label={t("team-add-member-dialog-checkbox-admin")}
-        />
+      <DialogContent>
+        <Box display="flex" flexDirection="column">
```

Comment:
The drawback adding a box, is it add an unnecessary div element to the page. But why not.